pkg/api/resource/quantity.go

```diff
@@ -112,10 +112,19 @@ func MustParse(str string) Quantity {
 	return *q
 }
 
+// Scale is used for getting and setting the scaled value.
+type Scale int
+
+const (
+	Nano  Scale = -9
+	Micro Scale = -6
+	Milli Scale = -3
```

Comment:
I think the comment addressed this, but really the only reason is it would complicate the implementation and there doesn't seem to be a big enough usecase for it.